datalake/src/main/java/com/sequenceiq/datalake/flow/delete/SdxDeleteActions.java

```diff
@@ -193,4 +202,11 @@ protected Object getFailurePayload(SdxDeletionFailedEvent payload, Optional<SdxC
             }
         };
     }
+
+    private void notifyEvent(SdxContext context, ResourceEvent resourceEvent) {
```

Comment:
Anyway to centralize this method somewhere else? Seems like you use it in various places with minimal differences.